jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/sso/Authorization.kt

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.credentials.sso
+
+import java.time.Instant
+
+data class Authorization internal constructor(
+    val deviceCode: String,
+    val userCode: String,
+    val verificationUri: String,
+    val verificationUriComplete: String,
+    val expiresAt: Instant,
+    val pollInterval: Long
```

Comment:
Can you signify that this is in seconds somehow? Either by name or type `Duration`?

Reply:
Done